### ISSUE TITLE
System.Guid apparently not supported

### ISSUE THREAD

**Issue by claorden** [tehmantra/saunter#17]
In my many of my classes I have properties of type "System.Guid". This type has itself an Empty property of type "System.Guid". Hence, the component generated by Saunter is:

~~~
      "guid":{
            "properties":{
               "Empty":{
                  "$ref":"#/components/schemas/guid"
               }
            }
         }
~~~

When I try to render the resulting json file in the [AsynApi editor](https://playground.asyncapi.io/?utm_source=editor&utm_medium=editor-topbar&utm_campaign=playground) it throws the error "undefined". This could be an infinite loop. Simply setting guid to type string solves the problem, but it is not the desired behaviour.

Maybe I'm doing something wrong?

**Comment by tehmantra** [comment]
This is a bug.
From investigating, looks like I need to add another case here: 
https://github.com/tehmantra/saunter/blob/0e67589dcb0da01202375a53f97c2602d815b711/src/Saunter/Generation/SchemaGeneration/SchemaGenerator.cs#L128-L175

```csharp
if (type.IsGuid())
{
    return new Schema
    {
        Type = "string",
        Format = "uuid",
    };
}
```

I'll get this fixed.

**Comment by claorden** [comment]
That was fast! Thank you very much @tehmantra

**Comment by tehmantra** [comment]
Fixed in v0.2.0: https://www.nuget.org/packages/Saunter/0.2.0
